Add documentation for running magics in remote kernels (#196)
jupyterlab/jupyter-ai: docs/source/users/index.md

## The `%%ai` magic command

Jupyter AI can also be used in notebooks via Jupyter AI magics. This section
provides guidance on how to use Jupyter AI magics effectively. The examples in
this section are based on the [Jupyter AI example notebooks](https://github.com/jupyterlab/jupyter-ai/blob/main/examples/).

If you already have `jupyter_ai` installed, the magics package
`jupyter_ai_magics` is installed automatically. Otherwise, run

    pip install jupyter_ai_magics

in your terminal to install the magics package.

Before you send your first prompt to an AI model, load the IPython extension by
running the following code in a notebook cell or IPython shell:

```
%load_ext jupyter_ai_magics
```

This command should not produce any output.

:::{note}

> If you are using remote kernels (e.g.  Amazon SageMaker Studio), the above
> command will throw an error. This means that need to install the magics package
> on your remote kernel separately, even if you already have `jupyter_ai_magics`
> installed in your server's environment. In a notebook, run

Reviewer: ```suggestion
If you are using remote kernels (e.g.  inside Amazon SageMaker Studio), you need to install the magics package on your remote kernel separately. In a notebook, run
```

Author: Resolved per internal discussion.
